src/uct/ib/rc/accel/rc_mlx5_common.h

```diff
@@ -122,21 +128,29 @@ enum {
 
 /* TODO: Remove/replace this enum when mlx5dv.h is included */
 enum {
-    UCT_RC_MLX5_OPCODE_TAG_MATCHING        = 0x28,
-    UCT_RC_MLX5_CQE_APP_TAG_MATCHING       = 1,
+    UCT_RC_MLX5_OPCODE_TAG_MATCHING          = 0x28,
+    UCT_RC_MLX5_CQE_APP_TAG_MATCHING         = 1,
+
+    /* last packet flag for multi-packet RQs */
```

Comment:
maybe follow the previous style of curent enum declaration?
```
...

/* multi-packet RQs fields */
UCT_RC_MLX5_MP_RQ_LAST_MSG_FIELD           = 0x40000000,        /* last packet flag for multi-packet RQs */
UCT_RC_MLX5_MP_RQ_BYTE_CNT_FIELD_MASK = 0x0000FFFF,
UCT_RC_MLX5_MP_RQ_NUM_STRIDES_FIELD_MASK = 0x3FFF0000, /* byte count mask for multi-packet RQs */

 /* tag segment flags */
...
```

Reply:
lines would be too long then. Also other comments in this enum are made above elements